examples/discovery-search-app/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "baseUrl": "http://localhost:3000/",
-  "supportFile": "cypress/support/index.ts"
+  "supportFile": "cypress/support/index.ts",
+  "defaultCommandTimeout": 30000
```

Comment:
Why is this necessary?

Reply:
Because otherwise some cypress tests timeout before the page had the time to render.

Reply:
Then perhaps we need to either fix our tests or our code. I believe the default is `10000` (10s) -- that should be more than enough time. Please open an issue to fix this, so we can remove this line.

Also, @H-Fin created a smaller contract in order to cut down on how long tests take. Are we using that here?

Reply:
Probably not, since this was merged before that I believe. I can create an issue to fix this